ops/genesis/config.go

```diff
@@ -37,6 +37,7 @@ type GenesisConfig struct {
 	L2PredeployOwner        common.Address `json:"l2PredeployOwner"`
 	L1PortalAddress         common.Address `json:"l1PortalAddress"`
 	L1StandardBridgeAddress common.Address `json:"l1StandardBridgeAddress"`
+	L2BaseFeeRecipient      common.Address `json:"l2BaseFeeRecipient"`
```

Comment:
change this to `L2BaseFeeRecipientWallet` etc to avoid confusion w/ the "recipient fee vault" itself